app/assets/javascripts/data_grid.js.coffee

```diff
@@ -73,22 +107,22 @@ window.configs =
       [
         targets: [3]
```

Comment:
This numbers look cryptic. What do you think about aliasing them? Of course JS will not know if these aliases are correct but at least they will hint what was intended.